frontend/actions/actions.js

```diff
@@ -0,0 +1,20 @@
+const AppDispatcher = require("../dispatcher/dispatcher");
```

Comment:
@danielchangNYC Thanks for this tip!  I updated the webpack config to resolve.root and removed all those ugly "../"'s.  I also changed dispatcher.js to index.js to see if it defaulted to that file (which it did).  This is definitely something I will use moving forward.  Thanks again!